LayoutTests/imported/w3c/web-platform-tests/svg/sniffing-content-type.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>SVG Content Sniffing Test</title>
+    </head>
+    <body>
+        <div id="result"></div>
+        <script>
+            if (window.testRunner)
+                testRunner.dumpAsText();
```

Comment:
Since this is a WPT, can you use `testharness.js`/`testharness-vendor.js` and the [WPT test harness API](https://web-platform-tests.org/writing-tests/testharness-api.html) to generate the results?

Reply:
Fixed.